webrtc-stats.html

```diff
@@ -472,15 +473,27 @@ <h3>
             </p>
           </dd>
           <dt>
-            <dfn><code>track</code></dfn>
+            <dfn><code>sender</code></dfn>
```

Comment:
Should we have sender, receiver AND track? The sender/receiver referencing trackId.

In the sender case, you might expect counters to go up for the sender and for the current track, but as soon as you replaceTrack a different counter is used.

How useful are the track stats after we have sender/receiver stats? Which should be kept? @vr000m